### DISCUSSION TITLE
Creating a Zoomable/Expandable Sub-Process Object in BPMN

### DISCUSSION THREAD

**Discussion by BhanuT97** [clientIO/joint#2444] [Q&A]
### Introduction

I am working on a BPMN process and aiming to implement a zoomable sub-flow object. Specifically, I want the sub-process to expand and display the corresponding process details upon clicking the sub-flow object. Can anyone provide guidance or examples on how to achieve this using Joint.js? Your insights and assistance would be greatly appreciated.

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment]
There are several ways to approach this.
- open sub-flows in new tabs (https://www.jointjs.com/demos/tabs)
- to expand a sub-flow means to enlarge the container and display the children (https://www.jointjs.com/demos/container)
- infinite zoom (https://www.jointjs.com/demos/zoomable-user-interface-zui) - this experimental in _JointJS_
